project/Smithy4sBuildPlugin.scala

```diff
@@ -107,7 +107,9 @@ object Smithy4sBuildPlugin extends AutoPlugin {
     semanticdbVersion := scalafixSemanticdb.revision,
     testFrameworks += new TestFramework("weaver.framework.CatsEffect"),
     Test / fork := virtualAxes.?.value.forall(_.contains(VirtualAxis.jvm)),
-    Test / javaOptions += s"-Duser.dir=${sys.props("user.dir")}"
+    Test / javaOptions += s"-Duser.dir=${sys.props("user.dir")}",
+    // Ignores warnings in code using the deprecated Enum trait.
+    scalacOptions += "-Wconf:msg=object Enum in package api is deprecated:silent"
```

Comment:
FYI here's what Enum now generates as:

```scala
@deprecated(message = "The enum trait is replaced by the enum shape in Smithy 2.0", since = "2.0")
object Enum extends Newtype[List[EnumDefinition]] {
  val id: ShapeId = ShapeId("smithy.api", "enum")
  val hints : Hints = Hints(
    smithy.api.Documentation("Constrains the acceptable values of a string to a fixed set\nof constant values."),
    smithy.api.Trait(selector = Some("string :not(enum)"), structurallyExclusive = None, conflicts = None, breakingChanges = Some(List(smithy.api.TraitDiffRule(change = smithy.api.TraitChangeType.PRESENCE.widen, severity = smithy.api.TraitChangeSeverity.ERROR.widen, path = None, message = None)))),
    smithy.api.Deprecated(message = Some("The enum trait is replaced by the enum shape in Smithy 2.0"), since = Some("2.0")),
  )
  val underlyingSchema : Schema[List[EnumDefinition]] = list(EnumDefinition.schema).withId(id).addHints(hints).validated(smithy.api.Length(min = Some(1), max = None))
  implicit val schema : Schema[Enum] = recursive(bijection(underlyingSchema, asBijection))
}
```